src/Module.php

```diff
@@ -4,15 +4,19 @@
 
 namespace Laminas\Paginator;
 
-/** @final */
-class Module
+use Laminas\ServiceManager\ServiceManager;
+
+/**
+ * @psalm-import-type ServiceManagerConfiguration from ServiceManager
+ */
+final class Module
```

Comment:
Do we still need a `Module` class?

Reply:
Good point - we've dropped it elsewhere 👍